modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -24,5 +24,8 @@
   "log_correlation.header_config": "{'activityid':'Correlation-ID'}",
 
   "monitoring.jmx.rmi_server_start": false,
-  "product.version" : "${product.version}"
+  "product.version" : "${product.version}",
+
+  "transport.http.enabled" : true,
+  "transport.https.enabled" : true
```

Comment:
Do we need to allow disabling https?

Reply:
There are customers who run http only mode who terminates https in LB level. They might need this config.

Reply:
Can we make
 "transport.http.enabled" : false,

at defaults.json
Those who want to enable can do so. But always better to use https even for internal traffic.